resources/ext/cli/ca.erb

```diff
@@ -0,0 +1,30 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+require 'fileutils'
+require 'puppetserver/ca/stub'
+
+command = ARGV[0]
+
+unless command == "import"
+  STDERR.puts "  'import' is the only valid subcommand, try:"
+  STDERR.puts "    `puppetserver ca import <key> <bundle> <crl>"
+  exit 1
+end
+
+private_key = ARGV[1]
+cert_bundle = ARGV[2]
+crl_chain   = ARGV[3]
+
+unless private_key && cert_bundle && crl_chain
+  STDERR.puts "The import subcommand requires three arguments: private-key, cert-bundle, crl, in that order."
+  exit 1
+end
+
+ca_dir = '/etc/puppetlabs/puppet/ssl/ca'
+FileUtils.mkdir_p ca_dir
+Puppetserver::Ca::Stub.import(private_key, cert_bundle, crl_chain)
+FileUtils.touch(ca_dir + '/inventory.txt')
+File.open(ca_dir + '/serial', 'w+') {|f| f.puts '003' }
+FileUtils.chown_R 'puppet', 'puppet', ca_dir
```

Comment:
Do we ultimately want to move more of the logic in this file into the gem? What was the responsibility split you were envisioning between this wrapper and the gem?

Reply:
Yeah, I originally just put in the minimum into the gem, but needed to iterate on it for this, so I just put those workarounds in the script for the time being.

I was thinking the script, when we're done with this ticket and we bump the version of the gem we depend on, should have just the minimum to run the library. Something like:
```rb
require 'puppetserver/ca/config'
require 'puppetserver/ca/cli'

user_overrides = Puppetserver::CA::Config.parse(ARGV)
configuration  = Puppetserver::CA::Config.new(user_overrides)

Puppetserver::CA::CLI.run!(config)
```

Reply:
That looks like what I was expecting to see, 👍